src/applications/find-forms/widgets/createFindVaFormsPDFDownloadHelper/DownloadPDFGuidance.js

```diff
@@ -0,0 +1,65 @@
+// Dependencies.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+// relative imports
+import DownloadPDFModal from './DownloadPDFModal';
+import InvalidFormDownload from './InvalidFormAlert';
+import { onDownloadLinkClick, sentryLogger } from './index';
+import { showPDFModal } from '../../helpers/selectors';
+
+// DownloadPDFGuidance sole purpose is to render the react widget depending on params (PDF is valid or not)
+const DownloadPDFGuidance = ({
+  downloadUrl,
+  form,
+  formNumber,
+  formPdfIsValid,
+  formPdfUrlIsValid,
+  link,
+  netWorkRequestError,
+  store,
+}) => {
+  const div = document.createElement('div');
+  const parentEl = link.parentNode;
+
+  if (formPdfIsValid && formPdfUrlIsValid && !netWorkRequestError) {
+    // feature flag
+    if (store?.getState && showPDFModal(store.getState())) {
+      ReactDOM.render(
+        <Provider store={store}>
+          <DownloadPDFModal formNumber={formNumber} url={downloadUrl} />
+        </Provider>,
+        div,
+      );
+      parentEl.insertBefore(div, link);
```

Comment:
I think this could be done with a portal as well, but either way works.

Reply:
https://reactjs.org/docs/portals.html

Reply:
Oo, going to check this out.

Reply:
Sorry about that! I was requesting changes for the copy change, I'm cool with this 👍